docs/about-stacks.md

```diff
@@ -24,47 +24,44 @@ A terraform stack is a runnable terraform module that operates on a subset of
 the infrastructure's resources.
 ```
 
-By _runnable_ it means it has everything needed to call
-`terraform init/plan/apply` . For example, a module used by other modules, ie,
-don't have the `provider` explicitly set, is not runnable hence it's
-**not a stack**.
+For a module to be considered a stack, it must be complete and self-contained, 
+meaning that it can run all the necessary Terraform commands (`init/plan/apply`) 
+without any external dependencies.
 
-If your runnable Terraform module creates your whole infrastructure, *it's
-also not a stack*, since the idea of stacks is to be a unit of infrastructure
-decomposition.
+If a module is missing any required components, such as provider settings, 
```

Comment:
Here I would also say that we explicitly list mandatory components such as provider settings and optional components such as state